packages/react/src/addresses/hooks/useAddresses.ts

```diff
@@ -43,14 +56,52 @@ export interface MetaData {
  */
 export default ({ userId }: MetaData): any => {
   // Selectors
-  const addressesData = useSelector((state: any) =>
+  const addressesData = useSelector((state: StoreState) =>
     selectors.getAddresses(state),
   );
-  const addressesIds = useSelector((state: any) => selectors.getResult(state));
-  const addressesError = useSelector((state: any) => selectors.getError(state));
-  const isAddressesLoading = useSelector((state: any) =>
+  const addressesIds = useSelector((state: StoreState) =>
+    selectors.getResult(state),
+  );
+  const addressesError = useSelector((state: StoreState) =>
+    selectors.getError(state),
+  );
+  const isAddressesLoading = useSelector((state: StoreState) =>
     selectors.isAddressesLoading(state),
   );
+  const addressError = useCallback((addressId: string) => {
+    useSelector((state: StoreState) =>
+      selectors.getAddressError(state, addressId),
+    );
+  }, []);
+  const isAddressLoading = useCallback((addressId: string) => {
+    useSelector((state: StoreState) =>
+      selectors.isAddressLoading(state, addressId),
+    );
+  }, []);
```

Comment:
In order for the useCallback to work properly you should add the properties that when changed will fetch the new value instead of opting for the memoized one. So in this case you should add the address id in the second argument.

Reply:
After discussion in Slack the addressId is only provided in the function usage so the hook doesn't have access to it, marking this as solved.